llama_stack/providers/utils/memory/vector_store.py

```diff
@@ -145,6 +145,10 @@ async def query(
     ) -> QueryDocumentsResponse:
         raise NotImplementedError()
 
+    @abstractmethod
+    async def delete(self):
```

Comment:
is this used anywhere?

Reply:
yes, this is the parent class of all the indexes